docs/dev/05-plugins.md

```diff
@@ -77,4 +78,9 @@ statuses. The method takes an object of the form:
 
     skipped: Boolean // skipped / ran
 }
+
+type SpecNames = {
+    '_': Array[String],
+    [$subSpec]: SpecNames
+}
 ```
```

Comment:
I think you could add a paragraph at the end something like:

In general the `window.__karma__` callbacks communicate their data to karma reporters.  Functions like `info` can add information that will be passed to the reporter; some reporters may ignore some data from the callbacks.